sources/packages/web/src/views/aest/student/Overawards.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <p class="header-sub-title mt-9">Overawards</p>
```

Comment:
it's not matching the other tabs 
![image](https://user-images.githubusercontent.com/77353155/221010539-ffcf05d7-d605-463e-a0dd-ac946e8e56cb.png)
![image](https://user-images.githubusercontent.com/77353155/221010678-66768484-dbc0-4ff7-8e1f-36da8a8bf35b.png)

Reply:
The notes title is not correct as per figma.

This is how it is 

![image](https://user-images.githubusercontent.com/54600590/221034684-57c1d7c2-7d19-4427-9a3f-01070fd85e7a.png)

Reply:
mm... I think figma may have updated after the latest refactor. but we follow the format, right? in the notes there is a `div` container